src/math/SpatialCell.cc

```diff
@@ -86,9 +86,8 @@ SpatialCell::SpatialCell(std::string const& label, ///< string representing "nam
     _candidateList(candidateList),
     _ignoreBad(true)
 {
-    lsst::pex::logging::TTrace<3>("lsst.afw.math.SpatialCell",
-                                  "Cell %s : created with %d candidates",
-                                  this->_label.c_str(), this->_candidateList.size());
+    LOGL_DEBUG("TRACE2.afw.math.SpatialCell", "Cell %s : created with %d candidates",
+               this->_label.c_str(), this->_candidateList.size());
```

Comment:
I did find some exceptions: we should keep the `TRACE#` prefixes for `afw.math.convolve` and `afw.math.warp` loggers, since I think it's conceivable they could be directly enabled and they contain multiple levels.